ext/opentelemetry-ext-opentracing-shim/tests/testbed/test_nested_callbacks/README.rst

```diff
@@ -0,0 +1,46 @@
+
+Nested callbacks example.
+=========================
+
+This example shows a ``Span`` for a top-level operation, and how it can be passed down on a list of nested callbacks (always one at a time), have it as the active one for each of them, and finished **only** when the last one executes. For Python, we have decided to do it in a **fire-and-forget** fashion.
+
+Implementation details:
+
+
+* For ```asyncio`` and ``threading``\ , the ``Span`` is manually passed down the call chain, activating it in each corotuine/task.
+* For ``tornado``\ , the active ``Span`` is not passed nor activated down the chain as the  ``Context`` automatically propagates it.
```

Comment:
is this true? by the looks of the example, it seems that the span is being passed along by directly referencing it inside of a closure, which I think is a valid strategy regardless of the execution engine.

Reply:
I *think* this is a rusty comment from OT, and thus not updated ;)

Reply:
For the threading case it is needed to activate it manually while in asyncio and tornado (that is using contextvars) it is done automatically. The comment was wrong.

Reply:
I updated the comment removing the part "passing" part, it is true that in all cases it's passing by referencing in a closure, the difference is that in the threads case it has to be manually activated.